css/style.css

```diff
@@ -1,3 +1,522 @@
+  height: 9px;
+  transition: transform 1s linear;
+}
+
+.product-camel_g:hover {
+  transform: scale(1.2, 1.2);
+}
+
+.product-camel_o {
+  position: absolute;
+  display: inline-block;
+  right: 75px;
+  bottom: 20px;
+  background: #E29D69;
+  border-radius: 50%;
+  width: 9px;
+  height: 9px;
+  transition: transform 1s linear;
+}
+
+.product-camel_o:hover {
+  transform: scale(1.2, 1.2);
+}
+
+.product-camel_r {
+  position: absolute;
+  display: inline-block;
+  right: 55px;
+  bottom: 20px;
+  background: #B62E3C;
+  border-radius: 50%;
+  width: 9px;
+  height: 9px;
+  transition: transform 1s linear;
+}
+
+.product-camel_r:hover {
+  transform: scale(1.2, 1.2);
+}
+
+.product-camel_y {
+  position: absolute;
+  display: inline-block;
+  right: 35px;
+  bottom: 20px;
+  background: #FDB431;
+  border-radius: 50%;
+  width: 9px;
+  height: 9px;
+  transition: transform 1s linear;
+}
+
+.product-camel_y:hover {
+  transform: scale(1.2, 1.2);
+}
+
+/*mandel*/
+
+.products__product-mandel {
+  align-self: flex-start;
+  text-align: left;
+  position: relative;
+  width: 280px;
+  height: 340px;
+}
+
+.products__product-mandel::after {
+  content: "";
+  border-right: 1px solid #CCCCCC;
+  position: absolute;
+  width: 280px;
+  height: 130px;
+  top: 60px;
+}
+
+.product-mandel_g {
+  position: absolute;
+  display: inline-block;
+  right: 95px;
+  bottom: 20px;
+  background: #C4C4C4;
+  border-radius: 50%;
+  width: 9px;
+  height: 9px;
+  transition: transform 1s linear;
+}
+
+.product-mandel_g:hover {
+  transform: scale(1.2, 1.2);
+}
+
+.product-mandel_b {
+  position: absolute;
+  display: inline-block;
+  right: 75px;
+  bottom: 20px;
+  background: #762E31;
+  border-radius: 50%;
+  width: 9px;
+  height: 9px;
+  transition: transform 1s linear;
+}
+
+.product-mandel_b:hover {
+  transform: scale(1.2, 1.2);
+}
+
+.product-mandel_bl {
+  position: absolute;
+  display: inline-block;
+  right: 55px;
+  bottom: 20px;
+  background: #41479B;
+  border-radius: 50%;
+  width: 9px;
+  height: 9px;
+  transition: transform 1s linear;
+}
+
+.product-mandel_bl:hover {
+  transform: scale(1.2, 1.2);
+}
+
+.product-mandel_y {
+  position: absolute;
+  display: inline-block;
+  right: 35px;
+  bottom: 20px;
+  background: #FDB431;
+  border-radius: 50%;
+  width: 9px;
+  height: 9px;
+  transition: transform 1s linear;
+}
+
+.product-mandel_y:hover {
+  transform: scale(1.2, 1.2);
+}
+
+/*cloud*/
+
+.products__product-cloud {
+  align-self: flex-start;
+  text-align: left;
+  position: relative;
+  width: 280px;
+  height: 340px;
+}
+
+.product-cloud_g {
+  position: absolute;
+  display: inline-block;
+  right: 95px;
+  bottom: 20px;
+  background: #C4C4C4;
+  border-radius: 50%;
+  width: 9px;
+  height: 9px;
+  transition: transform 1s linear;
+}
+
+.product-cloud_g:hover {
+  transform: scale(1.2, 1.2);
+}
+
+.product-cloud_b {
+  position: absolute;
+  display: inline-block;
+  right: 75px;
+  bottom: 20px;
+  background: #257DA9;
```

Comment:
Codacy found an issue: [Expected "#257DA9" to be "#257da9" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/eyurodin_axion/pullRequest?prid=5738041)